src/main/javascript/software.bytepushers.utils.DateUtility.js

```diff
@@ -27,5 +27,28 @@
         }
         return 0;
     };
+    /*checks if string passed in is valid MM/DD/YYYY date*/
+    BytePushers.DateUtility.isDateString_MMDDYYYY = function (dateString) {
+        /*credit: https://stackoverflow.com/questions/6177975/how-to-validate-date-with-format-mm-dd-yyyy-in-javascript/6178341#6178341*/
+
+        /* First check for the pattern */
```

Comment:
Is line 34 through 52 used somewhere else.  We might consider breaking down into smaller re-usable methods.